NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
Is this file important?

Reply:
VS adds a nuget.config file automatically. Is this OK or does something need fixing @Happypig375 ?

Reply:
I've removed it from this PR so we figure out what to do later when it crops up again.